numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -1222,7 +1225,7 @@ def add_newdoc(place, name, doc):
            https://en.wikipedia.org/wiki/Exponential_function
     .. [2] M. Abramovitz and I. A. Stegun, "Handbook of Mathematical Functions
            with Formulas, Graphs, and Mathematical Tables," Dover, 1964, p. 69,
-           http://www.math.sfu.ca/~cbm/aands/page_69.htm
+           https://personal.math.ubc.ca/~cbm/aands/page_69.htm
```

Comment:
Should the other reference links be changed to include the page number?

Reply:
For this PR, I'm just fixing the existing links.  I wasn't planning on digging into the individual topics to find the appropriate pages.

Reply:
Most of the references have a page number, just not in the link. The argument for making the fix now is that it will otherwise slip into the background and get lost.

Reply:
> Most of the references have a page number, just not in the link.

OK, you've kicked me out of mindless substitution mode :grin: .  I'll add the page number links.

Reply:
Done.